README.md

```diff
@@ -411,6 +426,14 @@ function handleRoute (req, res, ctx, done) {
 }
 ```
 
+### app.use(plugin)
+Register a plugin on the merry instance. A plugin is an object where the
+following values are possible:
```

Comment:
there is only one value so far, so maybe rephrase to `where this is possible` or something like that?